### ISSUE TITLE
don't work ? 

### ISSUE THREAD

**Issue by LastKing** [jkuri/ng2-datepicker#70]
can you give me a example? thanks !!
![image](https://cloud.githubusercontent.com/assets/8540374/19217927/14d6cce2-8e1c-11e6-8bc3-99735e4b17a5.png)

**Comment by ghetolay** [comment]
Can we have the complete error please ?

Probably same as #66.

**Comment by LastKing** [comment]
Ok
1.first install ng-datepicker

2.add link to index.html
![8kp4fy4h yc fthj h4i1vl](https://cloud.githubusercontent.com/assets/8540374/19373147/72bbe528-91f4-11e6-8526-c938b4bdcb3b.png)

3.import model
![pl x unr_ 0c4v2_h ate8](https://cloud.githubusercontent.com/assets/8540374/19373186/c600967a-91f4-11e6-96e1-c9e6b2e001a1.png)

4.I  use to webpack,no system.js
`module.exports = {
  debug: true,
  devtool: 'cheap-module-source-map',
  entry: {
    vendor: ['../web/vendor.ts', 'webpack/hot/dev-server', 'webpack-hot-middleware/client'],
    main: ['../web/main.ts', 'webpack/hot/dev-server', 'webpack-hot-middleware/client']
  },

  output: {
    path: '/',
    publicPath: '/bundles/',
    filename: '[name].js',
    chunkFilename: '[name].chunk.js'
  },

  resolve: {
    extensions: ['', '.ts', '.js', '.json', '.css']
  },

  module: {
    loaders: [
      {
        test: /\.ts$/,
        loaders: [
          'babel',
          'awesome-typescript-loader',
          'angular2-template-loader',
          'angular2-load-children-loader',
          '@angularclass/hmr-loader',   //启用热加载
        ],
        exclude: [/node_modules\/(?!(ng2-.+))/]
      },
      {test: /\.(woff|woff2)(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=application/font-woff'},
      {test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=application/octet-stream'},
      {test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: 'file'},
      {test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml'},
      {
        test: /\.css$/,
        loader: 'style!css',
        exclude: [/app/]
      },
      {
        test: /\.(html|css)$/,
        loader: 'raw-loader',
        include: [/app/]
      }
    ]
  },
  plugins: [
    new webpack.HotModuleReplacementPlugin(),
    new webpack.DefinePlugin({
      'process.env': {
        'NODE_ENV': '"development"'
      }
    }),
    new webpack.optimize.CommonsChunkPlugin('common.js', ['vendor', 'main'])
  ]
};`

I'm English is very bad. sorry..
Thank you for your help.

**Comment by ghetolay** [comment]
Well I wasn't asking for your setup but for the complete error.
On your first post you have a screenshot with errors but we can't see the complete error : 

`Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:3000/ng2-datepicker/ng2-datepicker(...)`

What's the end of that ? ( the `(...)` part ).

Also you say using webpack but the error says `SystemJS` it's weird.

**Comment by LastKing** [comment]
![m k1u4 l5ynneybxz mjfy3](https://cloud.githubusercontent.com/assets/8540374/19465757/f8fd7b14-9538-11e6-8d59-9506a549b54c.png)
I am unable to reproduce the last situation. This is the case. Is the picture full error?

**Comment by ghetolay** [comment]
Yep it's unable to find the html template, definitely duplicate of #66.
We're working on it.
